ispypsa_runs/development/ispypsa_inputs/ispypsa_config.yaml

```diff
@@ -24,14 +23,22 @@ network:
     #   "discrete_nodes": REZs are added as network nodes to model REZ transmission limits
     #   "attached_to_parent_node": REZ resources are attached to their parent node (sub-region or NEM region)
     rezs: discrete_nodes
-traces:
+temporal:
+  operational_temporal_resolution_min: 30
   # The path to the folder containing parsed demand, wind and solar traces. If set to ENV the path will be retrieved
   # from the environment variable "PATH_TO_PARSED_TRACES"
   path_to_parsed_traces: ENV
   year_type: fy
   start_year: 2025
   end_year: 2025
   reference_year_cycle: [2018]
+  # Representative weeks to use instead of full yearly temporal representation.
+  # Options:
+  #   "None": Full yearly temporal representation is used.
+  #   list[int]: a list of integers specifying weeks of year to use as representative. Weeks of year are defined as
+  #   full weeks (Monday-Sunday) falling within the year. For example, if the list is "[0]" the model will only use the
+  #   first full week of each modelled year.
+  representative_weeks: [0, 13, 26, 39]
```

Comment:
Could be done now or later, but I think this should be generalised since there may be many aggregation options. I think it could be organised as follows:

1. Create a field under `temporal` called `aggregation`
2. Under aggregation, you can then "select" an option and then provide arguments

Basically, keep as is but add under an aggregation subheading. This then allows different options/input data structures for various aggregation methods

So for example:
```yaml
temporal
...
  aggregation:
    representative_weeks: [0, 13, 26, 39]
```

Reply:
Also, while it's nice to be consistent with Python, should we use 1-indexing for things like the first week rather than 0 indexing?

Reply:
Good suggestions. Made both these changes.